broadcaster/lib/broadcaster/router.ex

```diff
@@ -7,13 +7,42 @@ defmodule Broadcaster.Router do
   plug(Corsica, origins: "*")
   plug(Plug.Logger)
   plug(Plug.Static, at: "/", from: :broadcaster)
+  plug(Plug.Static, at: "/admin", from: :broadcaster)
```

Comment:
Not sure about this one. I can also link assets with `/stream/stream.html` and `/stream/stream.js` :thinking:

Reply:
Ye, I don't like that. I have an idea: create a new router (like described in the comment above), and put the new `Plug.Static` there, but create an distinction as to from where the files are served (using the `:from` option), so the files at root are served from `priv/static/public` and the files for admin are served from `priv/static/admin` or something like this.

Start of the admin router would contain something like this:
```elixir
plug :authenticate
plug Plug.Static, at: "/admin", from {:broadcaster, "/admin"}
plug :match
plug :dispatch

# the routes and `authenticate` functions
```